plugins/package-managers/python/src/main/kotlin/Poetry.kt

```diff
@@ -80,7 +80,7 @@ class Poetry(
         return listOf(ProjectAnalyzerResult(project, packages))
     }
 
-    private fun inspectLockfile(lockfile: File): PythonInspector.Result {
+    private fun inspectLockfile(lockfile: File, scopeName: String): PythonInspector.Result {
```

Comment:
Nit: From Poetry-perspective this should probably be called `dependencyGroupName` as it's only the caller who decides to map this to ORT's scopes, but I'm fine with keeping the parameter name as-is.